internal/services/securitycenter/security_center_subscription_pricing_resource.go

```diff
@@ -63,6 +63,7 @@ func resourceSecurityCenterSubscriptionPricing() *pluginsdk.Resource {
 					"VirtualMachines",
 					"Arm",
 					"Dns",
+					"OpenSourceRelationalDatabases",
```

Comment:
confirmed via https://github.com/Azure/Microsoft-Defender-for-Cloud/blob/4abba727a370dea7b0af5435fab52dd783ea2154/Pricing%20%26%20Settings/Azure%20Policy%20definitions/Azure%20Defender%20Plans/ASC-AzureDefender-for-All.json#L28-L117